pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.9.44"
```

Comment:
Should I make change in this pr? @squidarth

Reply:
you don't need to -- also, it's more typical to say "0.9.44rc2". Also increment the patch version, this should technically be "0.9.45rc2", since 0.9.44 is already released